specification/workloads/resource-manager/Microsoft.Workloads/SAPVirtualInstance/preview/2023-10-01-preview/examples/sapapplicationinstances/SAPApplicationServerInstances_Create.json

```diff
@@ -0,0 +1,93 @@
+{
+  "parameters": {
+    "subscriptionId": "6d875e77-e412-4d7d-9af4-8895278b4443",
+    "resourceGroupName": "test-rg",
+    "sapVirtualInstanceName": "X00",
+    "applicationInstanceName": "app01",
+    "api-version": "2023-10-01-preview",
+    "body": {
+      "location": "westcentralus",
```

Comment:
Remove westcentralus from all examples as that is no longer a supported region for ACSS.

Reply:
done